### ISSUE TITLE
Wrong results for remainder ?

### ISSUE THREAD

**Issue by kaidokert** [RustCrypto/crypto-bigint#84]
Confused about why this isn't working. Am i breaking some assumption?

```
        for (n, d, q, r) in &[
            (27,13,2,1),
            (26,13,2,0),
            (14,13,1,1),
            (13,13,1,0),
            (12,13,0,12), // reminder returns = 0
            (1,13,0,1), // reminder returns = 0
        ] {
            let lhs = U64::from_u8(*n);
            let rhs = U64::from_u8(*d);
            let expected_q = U64::from_u8(*q);
            let expected_r = U64::from_u8(*r);
            let (quot,rem) = lhs.div_rem(&rhs).unwrap();
            assert_eq!(expected_q, quot);
            assert_eq!(expected_r, rem);
        }
```

**Comment by tarcieri** [comment]
cc @mikelodder7

**Comment by mikelodder7** [comment]
I’ll take a look

**Comment by tarcieri** [comment]
Fixed in #85. Will cut a v0.4.2 release shortly

**Comment by kaidokert** [comment]
Thanks for a quick fix ! I can do RSA now :)
